src/main/scala/com/datastax/sparkstress/ReadTask.scala

```diff
@@ -75,9 +104,19 @@ class FTSOneColumn(config: Config, sc: SparkContext) extends ReadTask(config, sc
 }
 
 /**
+  * SparkSQL Full Table Scan One Column
+  */
+class SparkSqlFTSOneColumn(config: Config, sc: SparkContext) extends ReadTask(config, sc) {
+
+  def run(): Unit = {
+    val colorCounts = sqlContext.sql("SELECT color FROM ks.tab").count
```

Comment:
You know I'm obsessive about standardization, lol. All the other ones are "val count" (aside from the errant "filterCounts").